sdk/communication/Azure.Communication.Email/CHANGELOG.md

```diff
@@ -9,7 +9,8 @@
 ### Bugs Fixed
```

Comment:
All the sections except for "Other Changes" will need to be removed, otherwise the pipeline with flag this.

Reply:
Updated